src/common/snippets/src/lowered/pass/brgemm_blocking.cpp

```diff
@@ -91,21 +91,9 @@ SpecificIterationHandlers BrgemmBlockingBase::get_k_loop_handlers(size_t work_am
     return get_default_blocking_loop_handlers(work_amount, block_size);
 }
 
-std::tuple<size_t, size_t, size_t> BrgemmBlockingBase::get_blocking_params(const ov::snippets::lowered::ExpressionPtr& brgemm_expr) {
-    const auto& in_0_desc = brgemm_expr->get_input_port_descriptor(0);
-    const auto& in_1_desc = brgemm_expr->get_input_port_descriptor(1);
-    const auto& out_desc = brgemm_expr->get_output_port_descriptor(0);
-
-    const auto& in_0_planar_dims = get_planar_vdims(in_0_desc->get_shape(), in_0_desc->get_layout());
-    const auto& in_1_planar_dims = get_planar_vdims(in_1_desc->get_shape(), in_1_desc->get_layout());
-    const auto& out_preordered_dims = get_preordered_vdims(out_desc->get_shape(), out_desc->get_layout());
-
-    const auto& m = *++out_preordered_dims.rbegin();
-    const auto& n = *out_preordered_dims.rbegin();
-    const auto& k0 = *in_0_planar_dims.rbegin();
-    const auto& k1 = *++in_1_planar_dims.rbegin();
-    size_t k = 0;
-    OPENVINO_ASSERT(utils::merge_dynamic_dim(k, k0, k1), "Brgemm input descriptors have incompatible K dimension value.");
+std::tuple<size_t, size_t, size_t> BrgemmBlockingBase::get_blocking_params(const ov::snippets::lowered::ExpressionPtr& brgemm_expr) const {
```

Comment:
As far as I understand, the base `get_blocking_params` implementation is not used anywhere. Maybe it's worth to make this method pure virtual then?

Reply:
Previously only BrgemmCPU uses the common logic in `get_blocking_params`. I thought that `BrgemmBaseBlocking` should have the implemented logic also (Otherwise this logic would be in `BrgemmCPUBlocking` and  `BrgemmBaseBlocking` would have pure virtual method). Because of that I decided to leave this logic there